app.json

```diff
@@ -15,5 +15,13 @@
       "value": "sentence",
       "required": false
     }
-  }
+  },
+  "addons": [
+    {
+      "plan": "heroku-postgresql",
+      "options": {
+        "version": "9.5"
```

Comment:
i think the default version on heroku now is 10

Reply:
Ok Thanks, I made the changes